src/content-cli-export.ts

```diff
@@ -13,8 +13,9 @@ export class Export {
             .option("-p, --profile <profile>", "Profile which you want to use to list packages")
             .requiredOption("--packageKeys <packageKeys...>", "Exports only given package keys")
             .option("--includeDependencies", "Include variables and dependencies", "")
+            .option("--excludeActionFlows", "Don't export action flows", "false")
```

Comment:
Can we make sure to cast this to boolean or leave it as empty string to avoid evaluating to true for string `"false"`?